definitions/npm/ramda_v0.26.x/flow_v0.104.x-/ramda_v0.26.x.js

```diff
@@ -1328,10 +1328,10 @@ declare module ramda {
   declare function minBy<T, V>(fn: (x: T) => V, x: T): (y: T) => T;
   declare function minBy<T, V>(fn: (x: T) => V, x: T, y: T): T;
 
-  declare function sortBy<T, V>(
-    fn: (x: T) => V,
-  ): (x: Array<T>) => Array<T>;
-  declare function sortBy<T, V>(fn: (x: T) => V, x: Array<T>): Array<T>;
+  declare var sortBy : {|
+    <A, B>(fn: (x: A) => B, xs: $ReadOnlyArray<A>): Array<A>,
```

Comment:
I would remove `B` and use `string | number` in place of it, since those are the only things we should reasonably be sorting on.